projects/centre_console/inc/drive_fsm.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "ebrake_tx.h"
+#include "fsm.h"
+#include "mci_output_tx.h"
+#include "relay_tx.h"
+#include "status.h"
+#include "stdbool.h"
+
+typedef enum {
+  DRIVE_STATE_NEUTRAL = 0,
+  DRIVE_STATE_PARKING,
+  DRIVE_STATE_DRIVE,
+  DRIVE_STATE_REVERSE,
+  NUM_DRIVE_STATES
+} DriveState;
```

Comment:
This is an EE, no?

Reply:
it's the destination states used in the drive_fsm...

Reply:
Very well. I would have imagined that the broadcasted drive states would be 1:1 with the destination states here. But if not, then this makes sense